tests/storage/cbt/utils.py

```
@@ -0,0 +1,486 @@
+    vm_instance_type_name: str,
+    runner_script_filename: str,
+    container_name: str,
+    params_env_name: str,
+    runner_params: dict[str, Any],
+    pod_name_suffix: str,
+    pod_role: str,
+    include_restore_work_volume: bool = False,
+) -> VirtualMachineForTests:
+    """Create a boot PVC, run a restore runner against backup storage, deploy the VM."""
+    restore_id = cbt_resource_id(name=restored_vm_name)
+    LOGGER.info(f"CBT {pod_role} {restored_vm_name}: boot_volume_name={boot_volume_name}")
```

Ruff (0.15.21)

[warning] Logging statement uses f-string

(G004)